xml/install_caasp_heat_templates.xml

```diff
@@ -571,6 +579,12 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
         Log in to each node and add the update repository.
        </para>
        <screen>&prompt.ardana;sudo zypper ar http://nu.novell.com/SUSE/Updates/SUSE-CAASP/3.0/x86_64/update/ caasp_update</screen>
+       <para>
+        Note the importance of having set a valid DNS server in your environment XML
+        file, as <literal>nu.novell.com</literal> must be reachable in order to
+        proceed with the update. (You may attempt <command>ping nu.novell.com</command>
```

Comment:
Feels clunky, suggested rewrite:

_<literal>nu.novell.com</literal> must be reachable in order to proceed with the update, ensure you set a valid DNS server in your environment XML file._